translations/it/LC_MESSAGES/messages.po

```diff
@@ -26,7 +26,7 @@ msgstr "Un percorso verso il successo"
 
 #: /views/amp-ads.html:249
 msgid "AMP Ad"
-msgstr ""
+msgstr "AMP Ad"
```

Comment:
Just to confirm, is that the Italian translation for "AMP Ad"? If it's not, please don't fill in it.  By default, if there's no string, the English is used.